Marlin/src/pins/pins_FORMBOT_TREX2+.h

```diff
@@ -143,15 +143,13 @@
 #define HEATER_BED_PIN     58
 
 #define FAN_PIN             9
-//#define FAN1_PIN            4
-
+#if(DISABLED(FILAMENT_RUNOUT_SENSOR))
+  #define FAN1_PIN            4 //Dedicated laser port, can be repurposed as fil runout
+#endif
```

Comment:
This needs better formatting. Attention to detail inspires confidence.

Reply:
Updated